internal/executor/executor.go

```diff
@@ -1,4 +1,4 @@
-package executable
+package executor
```

Comment:
thoughts: I see why you've renamed this package. However, I named it `executable` because I had planned to add support for go plugins also, which would require its own executor implementation. What do you recommend the package structure and names be to support the addition cleanly?